changelog.d/3476.added

```diff
@@ -0,0 +1,2 @@
+Windows BIOS PXE install (via syslinux pxelinux.0, linux.c32 and wimboot tftp/http)
+Windows BIOS PXE install (via grub2 grub.0 and wimboot tftp/http)
```

Comment:
According to my knowledge, we should only have a single line per changelog fragment, I think a comma-separated list inside the brackets is also okay. No need to split it over multiple lines.